Cargo.toml

```diff
@@ -33,13 +33,8 @@ vtab-arrow = ["vtab", "num"]
 vtab-full = ["vtab-excel", "vtab-arrow"]
 extensions-full = ["httpfs", "json", "parquet", "vtab-full"]
 buildtime_bindgen = ["libduckdb-sys/buildtime_bindgen"]
-modern-full = [
-    "chrono",
-    "serde_json",
-    "url",
-    "r2d2",
-    "uuid",
-]
+modern-full = ["chrono", "serde_json", "url", "r2d2", "uuid", "polars"]
+polars = ["dep:arrow2", "dep:polars", "dep:polars-core"]
```

Comment:
Is the dep: necessary? If not let's delete it

Reply:
Yes, it is. 
I set `arrow2`, `polars`, `polars-core` optional. We only turn them on when feature `polars` enabled.

Reply:
I mean `polars = ["arrow2", "polars", "polars-core"]` will also work? Or is it a better practise to add dep: ?

Reply:
`polars = ["arrow2", "polars", "polars-core"]` will not work because items in `[ ]` are features, not dependencies.
The use of `dep` is documented here: https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

Reply:
Then why the current calamine or chrono works? they are also dependencies

Reply:
But I get it, I think we can add dep here